cypress/e2e/tests/migration/applicationinventory/assessment/assess_review.test.ts

```diff
@@ -49,68 +52,100 @@ describe(["@tier1"], "Application assessment and review tests", () => {
 
         // Perform application review
         application.perform_review("low");
-        cy.wait(2000);
+        cy.wait(2 * SEC);
         application.verifyStatus("review", "Completed");
 
         // Delete application
         application.delete();
-        cy.wait(2000);
+        cy.wait(2 * SEC);
     });
 
     it(["@interop"], "Application assessment and review with medium risk", function () {
         // Navigate to application inventory tab and create new application
         const application = new Assessment(getRandomApplicationData());
         application.create();
         cy.wait("@getApplication");
-        cy.wait(2000);
+        cy.wait(2 * SEC);
 
         // Perform assessment of application
-        application.perform_assessment("medium", stakeholdersNameList);
-        cy.wait(2000);
+        application.perform_assessment(
+            "medium",
+            stakeholdersNameList,
+            null,
+            null,
+            legacyPathfinder
+        );
+        cy.wait(2 * SEC);
         application.verifyStatus("assessment", "Completed");
 
         // Perform application review
         application.perform_review("medium");
-        cy.wait(2000);
+        cy.wait(2 * SEC);
         application.verifyStatus("review", "Completed");
 
         // Delete application
         application.delete();
-        cy.wait(2000);
+        cy.wait(2 * SEC);
     });
 
     it("Application assessment and review with high risk", function () {
         // Navigate to application inventory tab and create new application
         const application = new Assessment(getRandomApplicationData());
         application.create();
         cy.wait("@getApplication");
-        cy.wait(2000);
+        cy.wait(2 * SEC);
 
         // Perform assessment of application
         application.perform_assessment("high", stakeholdersNameList);
-        cy.wait(2000);
+        cy.wait(2 * SEC);
         application.verifyStatus("assessment", "Completed");
 
         // Perform application review
         application.perform_review("high");
-        cy.wait(2000);
+        cy.wait(2 * SEC);
         application.verifyStatus("review", "Completed");
 
         // Delete application
         application.delete();
-        cy.wait(2000);
+        cy.wait(2 * SEC);
+    });
+
+    it("Application with multiple assessments", function () {
+        const application = new Assessment(getRandomApplicationData());
+        application.create();
+        cy.wait("@getApplication");
+        cy.wait(2 * SEC);
+
+        application.perform_assessment("high", stakeholdersNameList);
+        application.clickAssessButton();
+        cy.wait(2 * SEC);
+        cy.contains("tr", legacyPathfinder).find("button.retake-button").should("have.length", 1);
+
+        application.perform_assessment(
+            "high",
+            stakeholdersNameList,
+            undefined,
+            undefined,
+            cloudNative
+        );
```

Comment:
Could you add the following checks here ?
1)' Assessment' column shows 'Completed'.
2) 'Application Risk' field on the 'Details' tab on the app side drawer is populated after a successful assessment . This would require a new method in the application class.